lib/client-assets.php

```diff
@@ -478,11 +478,14 @@ function gutenberg_extend_wp_api_backbone_client() {
 	wp_add_inline_script( 'wp-api', $script );
 
 	// Localize the wp-api settings and schema.
-	$schema_response = rest_do_request( new WP_REST_Request( 'GET', '/wp/v2' ) );
+	$schema_response = rest_do_request( new WP_REST_Request( 'GET', '/' ) );
 	if ( ! $schema_response->is_error() ) {
 		wp_add_inline_script( 'wp-api', sprintf(
 			'wpApiSettings.cacheSchema = true; wpApiSettings.schema = %s;',
-			wp_json_encode( $schema_response->get_data() )
+			wp_json_encode( array(
+				'namespace' => '',
+				'routes' => $schema_response->get_data()[ 'routes' ],
```

Comment:
Per failing test:

```
Function array dereferencing is not present in PHP version 5.3 or earlier (PHPCompatibility.PHP.NewFunctionArrayDereferencing.Found)
```

Further, why do we need to do this change at all? Can we not just encode the data as we had done previously?

Reply:
We can, but the data has a different shape. Instead of getting an object with keys `( namespace, routes )`, we have keys `( name, description, url, home, gmt_offset, timezone_string, namespaces, authentication, routes, permalink_structure )`. Note the plural `namespaces`. Hence the sort-of-faking happening here. How do you suggest proceeding? We can indeed pass the entire data and handle it differently on the client.

Reply:
As far as I can recall, `withAPIData` really only cares about the `routes` property, and if we assume the schema response' `routes` includes all registered routes, all the better. Other keys can simply be ignored.

Reply:
I've tested this change both as-is and by reverting the encoding changes. Works great!

I can submit a separate PR, but thought it might be easier for someone with commit access to make the change in this PR and put the encoding stuff back in. Cheers!